NuGetPackageVerifier.json

```diff
@@ -8,7 +8,8 @@
                 "Exclusions": {
                     "DOC_MISSING": {
                         "lib/netstandard2.0/Microsoft.AspNetCore.Identity.UI.dll": "This is a library with packaged UI",
-                        "lib/netstandard2.0/Microsoft.AspNetCore.Identity.UI.Views.dll": "This assembly is generated and contains precompiled razor pages"
+                        "lib/netstandard2.0/Microsoft.AspNetCore.Identity.UI.Views.V3.dll": "This assembly is generated and contains precompiled razor pages",
```

Comment:
Should the be "... auto generated …" instead?

Reply:
You mean the message? It's really not important.